_posts/2023-09-07-r-project-sprint/r-project-sprint.Rmd

```diff
@@ -0,0 +1,194 @@
+control issues](https://www.bbc.com/news/uk-66637156){target="_blank"} and [train
+strikes](https://www.bbc.com/news/business-66479059){target="_blank"} caused delayed
+arrival for some.
+
+
+# During the event
+
+The issues addressed at the event fell into four broad categories:
+
+1. [Translation](https://contributor.r-project.org/r-project-sprint-2023/projects/translation/){target="_blank"} of messages, errors, and warnings.
+
+2. Improving [documentation](https://contributor.r-project.org/r-project-sprint-2023/projects/documentation/){target="_blank"}.
+
+3. Addressing bug reports, such as those related to the [`stats` package](https://github.com/r-devel/r-project-sprint-2023/issues/33){target="_blank"}.
+
+4. Enhancements, such as [logging visual outcomes from base graphics calls](https://github.com/r-devel/r-project-sprint-2023/issues/75){target="_blank"}. 
+
+# Knowledge requirements for contributions
+
+Depending on the task at hand, the technical requirements for contributions
+included:
+
+* Read R source code and understand function implementation (documentation and
+translation).
+
+* Read, modify, or write R source code (bugs and enhancements).
+
+* Read, modify, or write C source code (bugs and enhancements).
+
+# Workflows for contributions
+
+
+During the sprint, the progress on these items was tracked [https://github.com/r-devel/r-project-sprint-2023/issues](https://github.com/r-devel/r-project-sprint-2023/issues){target="_blank"}. Either the initial report or the final resolution of these items may be seen in the R Bug Tracking System, known as [Bugzilla](https://bugs.r-project.org/){target="_blank"}.
+
+Depending on the nature of the proposed change, the contributor may need
+to build R from source. The instructions regarding how to do so on your personal computing
+enviroment are outlined in the [R Patched and Development Versions Chapter](https://contributor.r-project.org/rdevguide/GetStart.html){target="_blank"} of the [R Development Guide](https://contributor.r-project.org/rdevguide/index.html){target="_blank"}. Alternatively, one could also build R from source without touching 
+their local computing environment using the [GitHub Code Space
+R Dev Containter](https://github.com/r-devel/r-dev-env){target="_blank"} (which I
+heard facilitated collaboration nicely as well).
+
+Regardless of whether you needed to build R from source, proposed changes could then be tested on multiple computing platforms via a pull request to [https://github.com/r-devel/r-svn](https://github.com/r-devel/r-svn){target="_blank"},
+which mirrors the official base R server. After tests have passed, the contributor could 
+extract the diff from the pull request to submit a proposed solution via Bugzilla.
+
+Once a proposal is submitted,  member(s) of the R core team review. If the 
+proposed change modifies the code base, additional checks are run against
+all packages on CRAN (which there are currently ~20,000) to determine
```

Comment:
I think not all changes are checked against all of CRAN but rather a subset (which is still large).

Reply:
k i have asked heather if she wants to review as well, maybe she can confirm